src/main/java/com/artipie/UsersFromStorageYaml.java

```diff
@@ -163,14 +163,14 @@ public UsersFromStorageYaml(final Storage storage, final Key key) {
      *  figure out why, make necessary corrections and use `ContentAs` here.
      */
     public CompletionStage<YamlMapping> yaml() {
-        return new RxStorageWrapper(this.storage)
-            .value(this.key)
-            .flatMap(content -> new Concatenation(content).single())
-            .map(Remaining::new)
-            .map(Remaining::bytes)
-            .map(bytes -> Yaml.createYamlInput(new String(bytes, StandardCharsets.US_ASCII)))
-            .map(YamlInput::readYamlMapping)
-            .to(SingleInterop.get());
+        return SingleInterop.fromFuture(
+            new ConfigFile(this.key).valueFrom(this.storage)
+        ).flatMap(content -> new Concatenation(content).single())
+        .map(Remaining::new)
+        .map(Remaining::bytes)
+        .map(bytes -> Yaml.createYamlInput(new String(bytes, StandardCharsets.US_ASCII)))
+        .map(YamlInput::readYamlMapping)
+        .to(SingleInterop.get());
```

Comment:
@genryxy same concern here, I'd suggest to transform `CompletionStage` into result directly without using `Single`